File: in-motion-server/Post/IMS.Post.Models/Dto/Incoming/VideoMetaDataDto.cs
namespace IMS.Post.Models.Dto.Incoming;

public class VideoMetaDataDto
{
    public string BucketName { get; set; }
    public string BucketLocation { get; set; }
    public string Filename { get; set; }
    public string ContentType { get; set; }
    public string Type { get; set; }
}

File: in-motion-ios/inMotion/inMotion/Models/DTO/Post/Outgoing/VideoMetaDataDto.swift
//
//  VideoMetaDataDto.swift
//  inMotion
//
//  Created by Kamil Pietrak on 31/10/2023.
//

import Foundation

struct VideoMetaDataDto: Codable {
    public var bucketName: String
    public var bucketLocalization: String
    public var filename: String
    public var contentType: String
    public var type: String // TODO: Change type of this to enum
}
